### PR TITLE
XW-547 | 'Full site' option persistent per session

### PR DESCRIPTION
**Pull request by Warkot**
https://wikia-inc.atlassian.net/browse/XW-547

:email: carbon copy: @hakubo @justnpT @bkoval

### CODE REVIEW

File: front/scripts/mercury/utils/domain.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/// <reference path="../../baseline/mercury.d.ts" />
+
+module Mercury.Utils {
+	/**
+	 * @returns {string}
+	 */
+	export function getDomain(): string {
```
Comment (rogatty) [inline front/scripts/mercury/utils/domain.ts]: How about passing `hostname` as a param and writing a unit test?

Comment (hakubo) [inline front/scripts/mercury/utils/domain.ts]: +1

Comment (hakubo) [inline front/scripts/mercury/utils/domain.ts]: or if you want:
`getDomain(host = window.location.host)`

so you don't have to pass it all the time

File: front/scripts/mercury/utils/domain.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/// <reference path="../../baseline/mercury.d.ts" />
+
+module Mercury.Utils {
+	/**
+	 * @param {string} hostname
```
Comment (bognix) [inline front/scripts/mercury/utils/domain.ts]: Could you indicate that hostname is not required? Maybe something like this is correct:
```
@param {string} [hostname=window.location.hostname]
```

Comment (Warkot) [inline front/scripts/mercury/utils/domain.ts]: Indeed, it is! http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values Thanks!

**Comment by hakubo** [comment]
lgtmr

**Comment by Warkot** [comment]
lgtm**R** :question: :open_mouth:

**Comment by rogatty** [comment]
LGTM

**Pull request merged**